mitiq/observable/observable.py

```diff
@@ -34,6 +34,16 @@ def __init__(self, *paulis: PauliString) -> None:
     def from_pauli_string_collections(
         *pauli_string_collections: PauliStringCollection,
     ) -> "Observable":
+        """Creates an Observable from one or more PauliStringCollection
+            instances.
```

Comment:
```suggestion
        """Creates an ``Observable`` from one or more ``PauliStringCollection``
            instances.
```

Code should be formatted as code where possible (in some other places too through this PR).